exporter/exporterhelper/internal/queuebatch/persistent_queue.go

```diff
@@ -146,8 +153,15 @@ func (pq *persistentQueue[T]) initClient(ctx context.Context, client storage.Cli
 }
 
 func (pq *persistentQueue[T]) initPersistentContiguousStorage(ctx context.Context) {
-	riOp := storage.GetOperation(readIndexKey)
-	wiOp := storage.GetOperation(writeIndexKey)
+	// 1. Try to load from new consolidated metadata first
+	if err := pq.loadQueueMetadata(ctx); err == nil {
+		return
+	}
+
+	// 2. Fallback to legacy individual keys for backward compatibility
```

Comment:
Please add a removal date for this. I think to be safe we need probably 6months.